TheEthicalHackerCup/Assets/DefenseMinigames/Insider/Scripts/EndScene/EndGame.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class EndGame : MonoBehaviour
+{
+    private Button btn;
+    // Start is called before the first frame update
+    void Start()
+    {
+        btn = gameObject.GetComponent<Button>();
+        btn.onClick.AddListener(EndGameClicked);
+    }
+
+    void EndGameClicked()
+    {
+        SceneManager.LoadScene("MainScene");
```

Comment:
Should go to after action report with `GameManager.GetInstance().SwitchToAfterActionReportScene(message)`